aiida_fleur/workflows/scf.py

```diff
@@ -510,9 +510,9 @@ def change_fleurinp(self):
             dist = 0.0
             fleurmode.set_inpchanges({'itmax': itmax, 'minDistance': dist})
 
-        elif converge_mode == 'gw':
+        elif converge_mode == 'spex':
             dist = 0.0
-            fleurmode.set_inpchanges({'itmax': itmax, 'minDistance': dist, 'gw': 1})
+            fleurmode.set_inpchanges({'itmax': itmax, 'minDistance': dist, 'spex': 1})
             if 'settings' in self.inputs:
                 self.inputs.settings.append({'additional_retrieve_list': ['basis.hdf', 'pot.hdf', 'ecore']})
```

Comment:
I just noticed this if case when looking over the changes. This looks wrong since settings should be a Dict, right?

Reply:
Doesn't https://github.com/JuDFTteam/aiida-fleur/blob/9affcc1b902ee5af57fd6feed59aaaa71e2afcb3/aiida_fleur/tools/common_fleur_wf.py `get_inputs_fleur` convert it into Dict if it isn't one?

Reply:
Are these files actually always produced if `spex=1`. Then we could also move this into the FleurCalculation itself

Reply:
yes, these are always produced by `spex=1`

Reply:
And without HDF5 the calculation doesn't work, correct?
I will move this into the `FleurCalculation` tomorrow then 👍. I anyway have to update the support 2.0 branch since I did some work in develop

Reply:
yes, without HDF5 it doesn't work.